docs/dom-testing-library/api-helpers.md

```diff
@@ -185,6 +186,19 @@ console.log(getRoles(nav))
 // }
 ```
 
+## `shouldExcludeFromA11yTree`
+
+This function will compute if the given element should be excluded from the
+accessibility API by the browser. It implements every **MUST** criteria from
+https://www.w3.org/TR/wai-aria-1.2/#tree_exclusion with the exception of
```

Comment:
```suggestion
[Accessibility Tree spec](https://www.w3.org/TR/wai-aria-1.2/#tree_exclusion) with the exception of
```

Reply:
Thank you for the review. I've been c&p too many spec links these days and got tunnel vision.